tests/test_pbm_load.py

```diff
@@ -72,7 +72,7 @@ def test_load_works_p1_ascii(self):
         palette.validate()
 
     def test_load_works_p4_in_mem(self):
-        file = BytesIO(b"P4\n4 2\n\x55")
```

Comment:
My understanding is that this wouldn't be a legitimate example of a binary pbm. There should be two (half-padded) bytes for the two rows in this image